packages/react-search-ui/src/SearchProvider.js

```diff
@@ -10,29 +10,46 @@ import defaultA11yMessages from "./A11yNotifications";
  * The SearchProvider primarily holds a reference to the SearchDriver and
  * exposes it to the rest of the application in a Context.
  */
-const SearchProvider = ({ children, config }) => {
-  const [driver, setDriver] = useState(null);
+const SearchProvider = ({ children, config = {}, driver }) => {
```

Comment:
I changed this to accept a driver as a parameter mostly so we can pass in a mock driver in tests.